### PR TITLE
Reapply 34/device_outbox in 35/device_outbox_again.py since the schem…

### PR DESCRIPTION
**Pull request by NegativeMjark**
…a was bumped before it landed on develop

### CODE REVIEW

**Comment by erikjohnston** [comment]
LGTM

File: synapse/storage/schema/delta/35/device_outbox_again.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import os
+
+
+def run_create(cur, database_engine, *args, **kwargs):
+    try:
+        delta_dir = os.path.join(os.path.dirname(__file__), "..")
+        synapse.storage.prepare_database.executescript(
+            cur, os.path.join(delta_dir, "34", "device_outbox.sql")
+        )
+    except:
```
Comment (erikjohnston) [inline synapse/storage/schema/delta/35/device_outbox_again.py]: Can this only catch database exceptions please 0:-)

Comment (NegativeMjark) [inline synapse/storage/schema/delta/35/device_outbox_again.py]: What do database exceptions look like?
Do you have an example I can crib off?

**Comment by erikjohnston** [comment]
LGTM

**Comment by erikjohnston** [comment]
LGTM

**Pull request merged**